packages/gatsby-theme-newrelic/src/utils/createTessen.js

```diff
@@ -68,6 +72,8 @@ const tessenAction =
 
     const customerId = getCookie('ajs_user_id');
     const anonymousId = getCookie('ajs_anonymous_id');
+    const loggedIn =
+      getSavedLoggedInStatus(config.product) ?? (await checkIfUserLoggedIn());
```

Comment:
is it fine to use `config.product` here to populate the `NewRelic-Requesting-Services` header?